sdks/python/apache_beam/examples/streaming_wordcount_debugging.py

```diff
@@ -0,0 +1,179 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""An example to use assert_that to validate streaming wordcount.
+
+It includes:
+  - PrintFn (DoFn) to inspect element, window, and timestamp.
+  - AddTimestampFn (DoFn) to modify timestamps.
+  - assert_that via check_gbk_format and equal_to_per_window (matchers).
+"""
+
+from __future__ import absolute_import
+
+import argparse
+import logging
+import re
+
+import six
+
+import apache_beam as beam
+import apache_beam.transforms.window as window
+from apache_beam.examples.wordcount import WordExtractingDoFn
+from apache_beam.options.pipeline_options import PipelineOptions
+from apache_beam.options.pipeline_options import SetupOptions
+from apache_beam.options.pipeline_options import StandardOptions
+from apache_beam.testing.util import assert_that
+from apache_beam.testing.util import equal_to_per_window
+from apache_beam.transforms.core import ParDo
+
+
+class PrintFn(beam.DoFn):
+  """A DoFn that prints label, element, its window, and its timstamp. """
+  def __init__(self, label):
+    self.label = label
+
+  def process(self, element, timestamp=beam.DoFn.TimestampParam,
+              window=beam.DoFn.WindowParam):
+    # Log at INFO level each element processed. When executing this pipeline
+    # using the Dataflow service, these log lines will appear in the Cloud
+    # Logging UI.
+    logging.info('[%s]: %s %s %s', self.label, element, window, timestamp)
+    yield element
+
+
+class AddTimestampFn(beam.DoFn):
+  """A DoFn that attaches timestamps to its elements.
+
+  It takes a string of integers and it attaches to each of them
+  a timestamp of its same value."""
+  def process(self, element):
+    for elem in element.split(' '):
+      logging.info('Adding timestamp to: %s', element)
+      yield beam.window.TimestampedValue(elem, int(elem))
+
+
+def run(argv=None):
+  """Build and run the pipeline."""
+  parser = argparse.ArgumentParser()
+  parser.add_argument(
+      '--output_topic', required=True,
+      help=('Output PubSub topic of the form '
+            '"projects/<PROJECT>/topic/<TOPIC>".'))
+  parser.add_argument(
+      '--use_assert_that', action='store_true',
```

Comment:
`--use_assert_that` option is not used, please remove it.

Reply:
Oops... I left it behind when I was trying to make it work for the Dataflow Runner in combination with streaming_wordcount_it_test.py.